tutorials/modeling_2D.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Modeling and fitting 2D images using Gammapy\n",
+    "\n",
+    "## Prerequisites:\n",
+    " - To understand how a generel modelling and fiiting works in gammapy, please refer to the [analysis_3d tutorial](analysis_3d.ipynb)\n",
+    "\n",
+    "## Context:\n",
+    "Classically, images analysis in IACTs has been done using a ring background analysis. To understand how this works in gammapy, please refer to the [ring_background tutorial](ring_background.ipynb). Recently, with the availabilty of background IRFs with DL3 data, we can directly do spatial modelling in 2D. Of course, you can also extract maps using the ring background method and then use the same for modelling as described here.\n",
```

Comment:
I don't think you need to mention ring analysis here.

Reply:
The context here is that you want the determine the position and morphology of an object. To do so, you don't necessarily have to resort to a full 3D fitting but can perform a simple image fitting, in particular in an energy range where the PSF does not vary strongly or if you want to explore a possible energy dependance of the morphology.

Reply:
I would give the objective. Here localize a source and/or constrain its morphology.